blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ None
 Changes
 =======
 
+- Removed unused setting: ``cluster.graceful_stop.reallocate`` from code,
```

Comment:
the `from code, data dictionary and docs` part is imho not necessary. For a user I think it is more interesting that setting it to false didn't have the documented effect. (So that they know that nothing changes for them, other than that the setting can't be specified anymore)